cypresse2e/cypress/integration/e2eTests/joinNow/joinNow.spec.js

```diff
@@ -110,6 +152,23 @@ describe('FightPandemics Sign Up Page', () => {
 			joinNowButton.should('be.visible').and('have.attr', 'aria-disabled', 'false');
 		});
 
+		it('After giving correct email and password user clicking Join Now  Button', () => {
+			var emailField = joinNow.getEmailField();
+			emailField.should('be.visible').and('have.attr', 'name', 'email');
+			cy.generateRandomEmail(8,5).then((email) => {
+			  joinNow.getEmailField().type(email);
+		  });
+			var passwordField = joinNow.getPasswordField();
+			passwordField.should('be.visible').and('have.attr', 'name', 'password');
+			passwordField.type(emailAndPassword.dummySamplePassword);
+			var confirmPasswordField = joinNow.getConfirmPasswordField();
+			confirmPasswordField.should('be.visible').and('have.attr', 'name', 'confirmPassword');
+			confirmPasswordField.type(emailAndPassword.dummySamplePassword);
```

Comment:
@JayashreeSamantasinghar functions for passwordField() and confirmPasswordField() were created in this file